### ISSUE TITLE
Warning on news

### ISSUE THREAD

**Issue by atgrau** [Tustin/fortnite-php#32]
Hi, I'm not sure if this is an issue, but I got problems getting constants `SAVETHEWORLD `and `BATTLEROYALE` of news module. 

`<br />
<b>Warning</b>:  Use of undefined constant SAVETHEWORLD - assumed 'SAVETHEWORLD'`

Maybe adding Self before constants:

```
35        if ($type != Self::SAVETHEWORLD && $type != Self::BATTLEROYALE)
36            throw new \Exception("Only SaveTheWorld and BattleRoyale news are currently supported");
```
Solves the problem. #33

**Comment by atgrau** [comment]
Sure, sry :)

**Comment by NeVeR2TrY** [comment]
Same here

**Comment by Tustin** [comment]
Fixed with 00a6fdccf05bc8091133f5ed73b898ee8cd2db2f.
